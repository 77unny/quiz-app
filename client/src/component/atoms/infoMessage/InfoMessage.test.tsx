import 'jest-styled-components';

import { render } from '@testing-library/react';
import React from 'react';

import InfoMessage from './index';

describe('<InfoMessage />', () => {
  it('Default InfoMessage', () => {
    const { container } = render(<InfoMessage children="<span>랜덤</span> 난이도를 선택하셨습니다" />);

    expect(container.firstChild).toMatchSnapshot(`<span>랜덤</span> 난이도를 선택하셨습니다`);
  });
});
