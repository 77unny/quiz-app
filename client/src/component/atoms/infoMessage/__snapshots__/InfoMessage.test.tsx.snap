// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoMessage /> Default InfoMessage: <span>랜덤</span> 난이도를 선택하셨습니다 1`] = `
.c0 {
  margin-bottom: 2rem;
  --tw-text-opacity: 1;
  color: rgba(107,114,128,var(--tw-text-opacity));
}

.c0 span {
  font-weight: 700;
  --tw-text-opacity: 1;
  color: rgba(0,200,150,var(--tw-text-opacity));
}

<p
  class="c0"
>
  <span>
    랜덤
  </span>
   난이도를 선택하셨습니다
</p>
`;
